merchant-sdk/example-app/src/main/java/net/gini/android/merchant/sdk/exampleapp/upload/UploadViewModel.kt

```diff
@@ -73,11 +72,12 @@ class UploadViewModel(
         }
     }
 
-    private fun setDocumentForReview(documentId: String) {
-        viewModelScope.launch {
-            giniMerchant.setDocumentForReview(documentId)
-        }
-    }
+    // TODO EC-62: we don't expose the document anymore in the Merchant SDK and instead we pass the document id to the ReviewActivity
```

Comment:
This has been changed in EC-60, we set the document for review in the MerchantSDK, when instantiating the container fragment. I don't think the client needs to handle this step, do you think otherwise? 🤔

Reply:
Thank you, I will delete it then completely.